src/domain/ecoverse/ecoverse.resolver.queries.ts

```diff
@@ -77,10 +74,7 @@ export class EcoverseResolverQueries {
     return await this.ecoverseService.getUsers();
   }
 
-  @Roles(
-    RestrictedGroupNames.CommunityAdmins,
-    RestrictedGroupNames.EcoverseAdmins
-  )
+  @Roles(RestrictedGroupNames.Members)
```

Comment:
This will restrict only to the Members, not all levels. Roles need to be added explicitly.